pandokia/db.py

```diff
@@ -16,7 +16,7 @@
 # Yes, it works on strings with \0 in them.
 # Should we do iso-8859 or just wait to support unicode?
 
-re_star_x_star = re.compile('^\*[^*]*\*$')
+re_star_x_star = re.compile(r'^\*[^*]*\*$')
```

Comment:
This `re_star_x_star` never get referenced anywhere else in the repo.

Reply:
can you rip it out then? (tho I can imagine why you'd hesitate to do so)

Reply:
I don't want to tbh.

Reply:
I get that
👍